extensions/amp-analytics/0.1/requests.js

```diff
@@ -167,7 +167,7 @@ export class RequestHandler {
       } else {
         preUrl = appendEncodedParamStringToUrl(preUrl, extraUrlParamsStr);
       }
-      return baseUrlTemplatePromise.then(preUrl => {
+      return baseUrlTemplatePromise.then(() => {
```

Comment:
Bug! These are not equivalent!

Reply:
That's right, I believe this is a bug in the original code. Adding @zhouyx, who introduced this in #11837:

See https://github.com/ampproject/amphtml/pull/11837/files#diff-b951eee683ce1cc97c5e22f9abd32f11R169

Also see https://lgtm.com/projects/g/ampproject/amphtml/snapshot/a4d5dedaa4349a630d64ac4ffa41ed08b90f7efe/files/extensions/amp-analytics/0.1/requests.js?sort=name&dir=ASC&mode=heatmap&excluded=false

Reply:
Failed to remove unused in #11837. Had #12776 to fix that.

Reply:
Thanks. File reverted.

Reply:
@jridgewell @zhouyx PTAL